### DISCUSSION TITLE
vpy setting for STAB_STD

### DISCUSSION THREAD

**Discussion by animeojisan** [hooke007/mpv_PlayKit#386] [z_ENG_discuss]
Thanks for the great app!

Is the following code correct for STAB_STD?

```
import vapoursynth as vs
from vapoursynth import core
import k7sfunc as k7f

clip = video_in

#################
# option #
#################

clip = k7f.STAB_STD(input=?, vs_t=?)

clip.set_output()
```

It doesn't seem to be working.　I would be happy to learn how to write the correct code.

**Comment by hooke007** [comment]
Param `input` must be specified.
`vs_t` is usually not touched by common users.

In this case, it should be like
```
clip = k7f.STAB_STD(input=clip)
```

BTW, it's quite slow and the effects is not good.

**Reply by animeojisan** [reply]
Thank you for your response!
